tools/rsdl/dotnet/demo/Program.cs

```diff
@@ -0,0 +1,110 @@
+﻿using Newtonsoft.Json;
```

Comment:
Package name needs to be changed - it is `demo`

Reply:
change the folder name to demo, the name of the binary is not controlled by that (in .net)